doc/api/fs.md

```diff
@@ -2697,10 +2697,19 @@ changes:
 * `callback` {Function}
   * `err` {Error}
 
-Asynchronously rename file at `oldPath` to the pathname provided
+Asynchronously rename file or directory at `oldPath` to the pathname provided
 as `newPath`. In the case that `newPath` already exists, it will
-be overwritten. No arguments other than a possible exception are
-given to the completion callback.
+be overwritten. `fs.rename` can throw the below error:
+
+- On **Linux** when renaming directory exists and is NOT empty, it throws a
+  `ENOTEMPTY` error. 
+- On **Unix** when renaming directory exists and is NOT empty, it throws a
+  `ENOENT` error. 
+- On **Windows** regardless of renaming directory is empty, it throws a 
+  `EPERM` error.
+  
+No arguments other than a possible exception (as mentioned above) are given to 
```

Comment:
This just repeats line 2698 in a more complex written form, and it adds confusion. It says an exception is given to the completion callback, but it is an error that is given. Also, more errors than the few documented can occur, and the sentence reads to me as saying that only the errors documented above can be passed. Better, I think, to remove the sentence.